src/js/media/media.js

```diff
@@ -81,6 +75,15 @@ vjs.MediaTechController.prototype.initControlsListeners = function(){
   });
 };
 
+vjs.MediaTechController.prototype.removeControlsListeners = function () {
```

Comment:
I think there's a problem with this. It will remove the removeControlsListeners and activateControls listeners when `controlsdisabled` is triggered, but what if `controlsenabled` is triggered again? I think this will break that use case.
e.g.
- enable controls
- disable controls
- enable controls again